datanode/gateway/graphql/schema.graphql

```diff
@@ -5094,6 +5094,8 @@ type BatchProposal {
   state: ProposalState!
   "RFC3339Nano time and date when the proposal reached the network"
   datetime: Timestamp!
+  "Terms of the proposal for a batch proposal"
```

Comment:
Does this actually cover the terms of all the proposals in the batch? Or does batch terms do something else